src/preferences/dialog/dlgprefwaveform.cpp

```diff
@@ -334,6 +343,9 @@ void DlgPrefWaveform::slotResetToDefaults() {
     // Don't normalize overview.
     normalizeOverviewCheckBox->setChecked(false);
 
+    // Don't show minute markers.
+    overviewMinuteMarkersCheckBox->setChecked(false);
+
```

Comment:
I think this is a nice feature we should enable by default. If users don't like it, they can disable it again.